NuGet.config

```diff
@@ -19,6 +19,7 @@
     <add key="dotnet3.1-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
+    <add key="core-setup" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-a49e99c5/nuget/v3/index.json" />
```

Comment:
Remove this, we shouldn't be manually adding references to 3.1 darc feeds (especially random ones that don't correspond to the final build) - port this change to 5.0.3xx instead: https://github.com/dotnet/installer/commit/beafdfb94215e62ab01f3b1825b7e810ef04f150